src/build.py

```diff
@@ -840,6 +858,27 @@ def V8():
     CopyBinaryToArchive(os.path.join(V8_OUT_DIR, a))
 
 
+def Jsc():
+  buildbot.Step('JSC')
+  Mkdir(JSC_OUT_DIR)
+
+  proc.check_call(['xcrun', PREBUILT_CMAKE_BIN, '-Wno-dev',
+                   '..', '-G', 'Ninja',
+                   '-DCMAKE_BUILD_TYPE="Release"',
+                   '-DPORT=Mac',
+                   '-DENABLE_WEBASSEMBLY=ON'],
+                  cwd=JSC_OUT_DIR)
+  proc.check_call(['ninja', 'jsc'], cwd=JSC_OUT_DIR)
+  proc.check_call(['../Tools/Scripts/run-javascriptcore-tests',
```

Comment:
Yeah. I expect that it will because it's a separate thing that CMake just prepends. But I also need to comandeer the bot slave itself and try it (or just merge and see if it works) because unlike the bot slave, I actually do have a real install of XCode on my machine.